packages/sites/src/use-sites-list-sorting.tsx

```diff
@@ -160,6 +160,16 @@ function sortAlphabetically< T extends SiteDetailsForSorting >(
 ) {
 	const normalizedA = a.title.toLocaleLowerCase();
 	const normalizedB = b.title.toLocaleLowerCase();
+	const aIsRetired = 0 === a.title.indexOf( '(Retired)' );
```

Comment:
I also have P2s that have `(Retired)` at the end, one uses `[RETIRED]`, and some just have `RETIRED` with no brackets.

To match just a few more cases maybe it would be safe to do a case insensitive compare and also match anywhere position in the title?

Reply:
Is it possible that we also use "deprecated", "old," or other similar words for other sites? Maybe we could agree on one format and rename those sites, assuming we own them?

Reply:
@p-jackson @wojtekn As it turns out, most of the retired P2s don't have any usable data embedded in their site title 😭 2e6b5-pb

I created a new WP-CLI command to produce the list: D94712-code

I looked into filtering `bloginfo` to prefix `(Retired)` but realized that's a bad idea because the API probably doesn't have the concept of 'display' vs. 'raw' data (we'd only want to filter the displayed data, not raw data that would be put in an `<input>` or similar).

I think my next idea is to run a script that updates all of the site titles to include `(Retired)` at the beginning.

Reply:
> I think my next idea is to run a script that updates all of the site titles to include `(Retired)` at the beginning.

Asked about this: p1670505582472069-slack-C0EKK7DD4